tools/run_tests/build_python.sh

```diff
@@ -39,6 +39,46 @@ GRPCIO=$ROOT/src/python/grpcio
 GRPCIO_TEST=$ROOT/src/python/grpcio_test
 GRPCIO_HEALTH_CHECKING=$ROOT/src/python/grpcio_health_checking
 
+#
+# Dependency steps.
+#
+install_grpcio_deps() {
+  cd $GRPCIO
+  pip install -r requirements.txt
+}
+install_grpcio_test_deps() {
+  cd $GRPCIO_TEST
+  pip install -r requirements.txt
+}
+#
+# End dependency steps.
+#
+
+#
+# Install steps. Requires that the `pip` command is appropriate (i.e. that the
+# virtual environment has been activated).
+#
+install_grpcio() {
+  CFLAGS="-I$ROOT/include -std=c89" LDFLAGS=-L$ROOT/libs/$CONFIG GRPC_PYTHON_BUILD_WITH_CYTHON=1 pip install $GRPCIO
+}
+install_grpcio_test() {
+  pip install $GRPCIO_TEST
+}
+install_grpcio_health_checking() {
+  pip install $GRPCIO_HEALTH_CHECKING
+}
+#
+# End install steps.
+#
+
+# Cleans the environment of previous installations
+clean_grpcio_all() {
+  (yes | pip uninstall grpcio) || true
```

Comment:
"pip uninstall --yes" strikes me as a less drastic means to the same end as "yes | pip uninstall". Is it preferable or are my tastes and sensibilities missing something?